lefthook.yml

```
@@ -18,27 +18,58 @@ pre-commit:
 
+# Claude Code integration - runs after file edits
+post-tool-use-edit:
+  parallel: true
+  commands:
+    ktlint-format:
+      glob: "**/*.kt"
+      run: |
+        echo "🔧 Auto-formatting Kotlin file: {files}"
+        ./gradlew ktlintFormat --daemon --quiet
+        echo "✅ KtLint formatting completed"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)